Learning_Guide/Coaching.md

```diff
@@ -0,0 +1,61 @@
+
+### Context: Coach-queue appointments
+
+- Am I asking high-quality questions? Was I able to communicate without words like: thing, stuff, and this/that/those.
+- Can I quickly establish what I need from the learner to start helping them, and ask for it in a clear way?
+- Can I be confident in a situation where I am expected to help find a solution with little context?
+- How quickly can I determine if I am not able to help in a given situation and how do I respond?
+- Can I communicate with teams who may be having both intrapersonal and technical blocks, and am I able to help coach both programming concepts and better team patterns?
+
+### Context: Code reviews
+
+- Are the comments I leave actionable, specific, and kind?
+- Do I feel comfortable reviewing code when I don’t know 100% of what it is doing?
+- Am I missing common pitfalls when reviewing (i.e. poor indentation, confusing variable naming, not seeing when code could be easily refactored for readability)
+- Am I building confidence in my ability to analyze code that is not mine?
+- Are there major gaps in my own learning that are preventing me from giving a high-quality review and am I tracking those gaps as they are brought to my attention?
+- When I am skilled at a concept am I able to leave refactoring suggestions that consider optimization as well as readability?
+
+### Context: Escalation
+
+- Was I able to identify within 15 minutes if I needed to escalate?
+- Did I feel comfortable admitting I needed to escalate, and am I able to still be confident in following coach-queue appointments after escalating.
+- Was I able to ask high-quality questions of the learner/team so that I could describe to the SEP what the issue is, and where I found myself blocked.
+
+## Additional Tips
+
+- **Always prioritize participating in coaching sessions when you have escalated to an SEP**. This is one of the best opportunities to learn you may encounter. You attempted to solve a problem, and when you could not you get to immediately see how someone with a ton of experience solves the issue. If you have another appointment scheduled move it back so that you get to learn from the SEP.
+- **If a team does not know how to explain the situation to a degree you need to be helpful, focus on coaching them towards breaking down the problem into a more manageable chunk where they can describe the issue**. Three questions to consider are: What do you think the issue is? What have you tried so far? What would you expect if it worked?
+- **If you know you will need your regular afternoon coffee break, schedule it ahead of time, you can get booked up quickly.**
+- **Ask for SEPs or other coaches to review your code reviews.**
+- **Review code reviews, see what other folks are catching that you may not be.**
```

Comment:
The work review implies a `code review` and we don't ask learners to do code reviews anymore. We ask them to coach and we ask them to do "goal completion evaluations" AKA `/review`.